internal/resource_remote_file.go

```diff
@@ -252,22 +266,32 @@ func (v *remoteFileResource) Schema(ctx context.Context, req resource.SchemaRequ
 				MarkdownDescription: "Path to the file on the Freebox",
 				Required:            true,
 				Validators: []validator.String{
-					stringvalidator.LengthAtLeast(1),
+					models.FilePathValidator(),
 				},
 			},
 			"source_url": schema.StringAttribute{
 				MarkdownDescription: "VM ethernet interface MAC address",
```

Comment:
This looks wrong ; probably copy pasting

Reply:
It was already fixed in #13.

Reply:
Ah gotcha, my bad, I was looking at the file as a whole (as suggested instead of the diff) and completly missed that bit